StayFit/Tab Bar Controller/Recommended Workout Tab/RecommendedWorkoutViewController.m

```diff
@@ -132,7 +132,7 @@ -(void) gymBuddy{
            else {
                self.workoutArray = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
                self.filteredWorkoutArray = self.workoutArray;
-               //prints out the contents of the API for verification
+              //prints out the contents of the API for verification
```

Comment:
same here looks duplicate comments

Reply:
yes, it's just one comment per if-else statement. Should I remove the same comment from the else blocks and keep it only in the if block?